plugins/vbe/fit.c

```diff
@@ -0,0 +1,302 @@
+/* -*- Mode: C; tab-width: 8; indent-tabs-mode: t; c-basic-offset: 8 -*-
+ *
+ * Library for U-Boot Flat Image Tree (FIT)
+ *
+ * This tries to avoid using glib so that it can be used in other projects more
+ * easily
+ *
+ * Copyright (C) 2022 Google LLC
+ * Written by Simon Glass <sjg@chromium.org>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <libfdt.h>
```

Comment:
How much do you need to depend on this? Could we perhaps reimplement the fdt parsing in fwupd?

Reply:
No I don't think that is a good idea. This is a fairly substantial library, many years old with a huge arrange of security and other fixes and is widely used in other tools.